docs/getting-started-guides/vsphere.md

```diff
@@ -12,24 +12,24 @@ This page covers how to get started with deploying Kubernetes on vSphere and det
 
 ### Getting started with the vSphere Cloud Provider
 
-Kubernetes comes with a cloud provider for vSphere. A quick and easy way to try out the cloud provider is to deploy Kubernetes using [Kubernetes-Anywhere](https://github.com/kubernetes/kubernetes-anywhere).
-
-This page also describes how to configure and get started with the cloud provider if deploying using custom install scripts.
+Kubernetes comes with a cloud provider for vSphere. 
+vSphere Cloud Provider for Kubernetes allows Kubernetes Pods to use enterprise grade vSphere Storage.
 
 ### Deploy Kubernetes on vSphere
 
-To start using Kubernetes on top of vSphere and use the vSphere Cloud Provider use Kubernetes-Anywhere. Kubernetes-Anywhere will deploy and configure a cluster from scratch.
+To start using Kubernetes on top of vSphere and use the vSphere Cloud Provider use [Kubernetes-Anywhere](https://github.com/kubernetes/kubernetes-anywhere).
 
 Detailed steps can be found at the [getting started with Kubernetes-Anywhere on vSphere](https://git.k8s.io/kubernetes-anywhere/phase1/vsphere/README.md) page.
 
 ### vSphere Cloud Provider
 
-vSphere Cloud Provider allows using vSphere managed storage within Kubernetes. It supports:
+vSphere Cloud Provider allows using vSphere managed enterprise grade storage within Kubernetes. It supports:
+
+- Enterprise class services such as de-duplication and encryption with vSAN, QoS, high availability and data reliability
+- Policy based management at granularity of container volumes
+- Volumes, Persistent Volumes, Storage Classes,  dynamic provisioning of volumes and scalable deployment of Stateful Apps with StatefulSets
 
-1. Volumes
-2. Persistent Volumes
-3. Storage Classes and provisioning of volumes.
-4. vSphere Storage Policy Based Management for Containers orchestrated by Kubernetes.
+For more detail visit [vSphere Storage for Kubernetes Documentation](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/index.html)
 
 Documentation for how to use vSphere managed storage can be found in the [persistent volumes user guide](/docs/concepts/storage/persistent-volumes/#vsphere) and the [volumes user guide](/docs/concepts/storage/volumes/#vspherevolume).
```

Comment:
The links in this paragraph are broken.

Reply:
checked with @BaluDontu, links are accessible from the preview site and not broken - https://deploy-preview-5241--kubernetes-io-master-staging.netlify.com/docs/getting-started-guides/vsphere/